transcendence-app/src/game/validators/index.ts

```diff
@@ -0,0 +1,32 @@
+import { registerDecorator } from 'class-validator';
+
+const GAME_COMMANDS = [
+  'paddleMoveRight',
```

Comment:
It appears to me as if this `GAME_COMMANDS` should be a shared object across fe and be.

Reply:
As commented above, we’ll think about it 👍